src/modules/ipc.cpp

```diff
@@ -184,7 +195,12 @@ namespace modules {
       set_hook(m_initial);
     } else {
       m_current_hook = -1;
+
       m_output.clear();
+      if (m_label) {
+        m_label->reset_tokens();
```

Comment:
Here you also need replace the token again, otherwise the user will see a literal `%output%` on their bar.

In particular, everytime `m_output` changes, you have to reset and replace the tokens and then call `broadcast()`. It's probably best to do that in a separate function.